tests/functional/modules/test_zos_operator_func.py

```diff
@@ -81,11 +81,21 @@ def test_zos_operator_positive_verbose_with_full_delay(ansible_zos_module):
     )
     endmod = time.time()
     timediff = endmod - startmod
-    assert timediff > 4
-    for result in results.contacted.values():
-        assert result["rc"] == 0
-        assert result.get("changed") is True
-        assert result.get("content") is not None
+    if timediff < 4:
+        print("\n........testable delay failed (short): assertion commented for testing....\n")
```

Comment:
I'm not sure, but I think this is wrong: "commented for testing"... That looks like one of my notes, but I don't recall working on this module.

Reply:
@richp405 - i tracked it back to this change [here](https://github.com/ansible-collections/ibm_zos_core/blame/6bfe12c4ffd7a1243a2ebde590b10a8f20d3e8f2/tests/functional/modules/test_zos_operator_func.py) which was something you decided to do and I thought it was related to the timeout test, would have to go back into that PR and review again , screen shot included 
<img width="1257" alt="image" src="https://user-images.githubusercontent.com/25803172/180921347-45883dea-967e-4fcb-ac6e-f0f92e329c85.png">